### PR TITLE
Fixed bitwise for itemStatus

### PR DESCRIPTION
**Pull request by TokyoSU**
## todo

- [X] I have added a changelog entry to CHANGELOG.md file on the branch/fork (if it is an internal change then it is not needed) 
- [X] Pull request meets the Coding Conventions standards: https://github.com/MontyTRC89/TombEngine/blob/master/CONTRIBUTING.md#coding-conventions

## Links to issue(s) this pull request concerns (if applicable)

N/A

## Description of pull request 

Reported by @Lwmte 
The bitwise operation for item.Status dont make sense, so i removed that and remplaced it with correct value.

### CODE REVIEW

File: TombEngine/Objects/TR5/Entity/tr5_larson.cpp [hunk 1]
```diff
@@ -73,36 +77,28 @@ namespace TEN::Entities::Creatures::TR5
...
 			creature->MuzzleFlash[0].Delay--;
-
 		if (creature->MuzzleFlash[1].Delay != 0)
 			creature->MuzzleFlash[1].Delay--;
 
 		if (item->TriggerFlags)
 		{
 			if (CurrentLevel == 2)
 			{
-				item->Animation.IsAirborne = false;
```
Comment (Lwmte) [inline TombEngine/Objects/TR5/Entity/tr5_larson.cpp:92]: Why all these fields are replaced by `item->AIBits = AMBUSH;`? Same for cases below.

**Review by Lwmte — COMMENTED** [review]
Also please convert ItemInfo `Status` field to use `ItemStatus` enum instead of `int` :)

File: TombEngine/Objects/TR5/Entity/tr5_larson.cpp (hunk 1 above)
Comment (TokyoSU) [reply TombEngine/Objects/TR5/Entity/tr5_larson.cpp:92]: I ported TOMB5 code but it's still wont work because it's missing the last comment at the end.

**Comment by TokyoSU** [comment]
TO Check:
- [X] TR4 Dog
- [ ] TR1 Skateboard Kid
- [ ] TR2 Skidman
- [ ] TR2 Spear guardian
- [ ] TR2 Sword guardian
- [ ] TR3 Shiva
- [ ] TR4 Skeleton
- [ ] TR4 Mummy
- [ ] TR5 Larson
- [ ] TR4 Cog

**Pull request merged**